cypress/integration/inventory/create-edit-delete-material-types.spec.js

```diff
@@ -0,0 +1,32 @@
+import TestTypes from '../../support/dictionary/testTypes';
+import permissions from '../../support/dictionary/permissions';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import getRandomPostfix from '../../support/utils/stringTools';
+import InventorySettings from '../../support/fragments/inventory/inventorySettings';
+import MaterialTypesPane from '../../support/fragments/inventory/materialType/materialTypesPane';
+
+describe('ui-inventory: Create, edit, delete material types', () => {
+  let userId = '';
+  const materialTypeName = `autoTestMaterialType.${getRandomPostfix()}`;
+  const newMaterialTypeName = `autoTestMaterialType.${getRandomPostfix()}`;
+
+  before(() => {
+    cy.createTempUser([permissions.uiCreateEditDeleteMaterialTypes.gui]).then(userProperties => {
+      userId = userProperties.userId;
+      cy.login(userProperties.username, userProperties.password);
+      cy.visit(SettingsMenu.inventoryPath);
```

Comment:
may be you can start from visit of SettingsMenu.materialTypePath and this current is excess. Please, check

Reply:
Done